setup_test.go

```diff
@@ -276,6 +276,7 @@ type logger interface {
 
 func setupTestClient(t logger, options ...ClientOptionFunc) (client *Client) {
 	var err error
+	options = append(options, SetBasicAuth("elastic", "elastic"))
```

Comment:
I think we need to remove that line again, now that we have both versions via Docker Compose. I wonder why that didn't cause any problems in the tests.

Reply:
Ok, i will remove this today, but this line didn't caused the problems, because it's just additional headers for request, and plain OSS Elastic just ignored them.